scripts/controller.js

```diff
@@ -46,13 +46,14 @@ Controller.createMenuHtml = function() {
 
 /** @param {string} type */
 Controller.iconTypeClass = function(type){
-  for(let each of Data.menuItems) {
+  return Data.menuItems.reduce((prev, each) => {
```

Comment:
Reduce would be great here if we wanted to find the last occurrence of something.